update-test-group.sh

```diff
@@ -1,15 +1,15 @@
-# This script sets a group of devices to a specific commit,
-# currently it will look for devices in an app that have an environment variable called "TEST"
+#!/bin/sh
+#
+# This script sets a group of devices to a specific commit, currently it will
+# look for devices in an app that have an environment variable called "TEST"
 # these devices will then update to whatever commit is supplied as arg $1
 
 COMMIT_HASH=$1
-source resin.env
+. resin.env
 
 device_list=$(curl -X GET -H "Authorization: Bearer  $authToken" -H "Content-Type: application/json" "https://api.$BASE_URL/v2/device?\$expand=device_environment_variable%2Capplication&\$filter=application/app_name%20eq%20%27$APP_NAME%27%20and%20device_environment_variable/env_var_name%20eq%20%27TEST%27&\$select=id" | jq '.d[] | .id')
 
-array=($device_list)
-for i in "${array[@]}"
-do
+for i in $device_list; do
```

Comment:
I dry tested it, didn't test it for real :) If there is no reason in particular for using an array then this should be ok.

We should remove needless bashisms and stick to POSIX shell, as these scripts are used by end-users, using different shells on different operating systems. The extension reads ".sh" and not ".bash" which implies shell. A user might start it as "sh blah.sh" as opposed to "./blah.sh". He might need to do it if he has the text file open in an editor, as otherwise it would fail with: "-bash: ./a.sh: /bin/sh: bad interpreter: Text file busy".

Thus sticking to POSIX shell will save users some surprises.